.azurepipelines/Ubuntu-GCC5.yml

```diff
@@ -32,7 +32,7 @@ jobs:
     do_non_ci_build: false
     do_non_ci_setup: true
     do_pr_eval: true
-    packages: QemuQ35Pkg,QemuSbsaPkg
+    packages: QemuQ35Pkg,QemuSbsaPkg,QemuPkg

```

Comment:
Please have the file change exactly match the one in https://github.com/microsoft/mu_devops/pull/150. That way a file sync does not need to just move the package location in the list.

mu_tiano_platforms line:
https://github.com/microsoft/mu_devops/blob/8e13f695de7b9d6cc7d069dd4250c75368904bca/.sync/azure_pipelines/custom/mu_tiano_platforms/Ubuntu-GCC5.yml#L37